packages/nimble-components/src/anchor-tabs/specs/README.md

```diff
@@ -57,7 +57,7 @@ We cannot extend FAST's tabs control because it has baked into it the idea that
 -   _CSS Classes and CSS Custom Properties that affect the component_ - None
 -   _Slots_
     -   `start` - content placed to the left of tab list
-    -   `anchortab` - slot for anchor tab elements
+    -   `tab` - slot for anchor tab elements
```

Comment:
The one issue with changing this slot name from `"anchortab"` to `"tab"` is that it allows `nimble-tabs` to slot `nimble-anchor-tab` elements (and `nimble-anchor-tabs` to slot `nimble-tab` elements). It's certainly not expected to work, but it might work well enough to mask the client's mistake. One solution is to add a `tabSlotName` property to `Tabs` and `AnchorTabs`. Alternatively, we could check the slotted elements in the `tabsChanged` handler and throw an error, but that feels clunkier.

Reply:
I'll go ahead and implement your `tabSlotName` suggestion.